rkt/cli_apps.go

```diff
@@ -150,6 +155,89 @@ func (ae *appExec) Set(s string) error {
 	return nil
 }
 
+// appInjectVolume is for --inject-volume flags in the form of: --inject-volume source=PATH,target=PATH
+type appInjectVolume apps.Apps
+
+func (aam *appInjectVolume) Set(s string) error {
+	app := (*apps.Apps)(aam).Last()
+	if app == nil {
+		return fmt.Errorf("--inject-volume must follow an image")
+	}
+
+	p := strings.SplitN(s, ":", 2)
+	if len(p) != 2 {
+		return fmt.Errorf("must be SOURCE_PATH:TARGET_PATH")
+	}
+
+	volName, err := types.SanitizeACName(p[1])
```

Comment:
Hm, shouldn't we at least warn about the resulting `volName` being different from what we passed to command line? Something like:
```go
if volName != p[1] {
	stderr("Changed volume target %q to %q", p[1], volName)
}
```
or maybe just bail out if `p[1]` is not a valid ACName instead of sanitizing it. See `SanitizeACName` docs - "It should not be called without the user explicitly asking for a suggestion."